scenarios/failed-knotx-fragment/tasks.conf

```diff
@@ -0,0 +1,54 @@
+global.handler.fragmentsHandler.config {
+  tasks {
+    error-task {
+      action = fetch-incorrect-data
+      onTransitions._success {
+        action = te-pebble
+      }
+    }
+  }
+
+  actions {
+    fetch-incorrect-data {
+      factory = http
+      config.endpointOptions {
+        path = /service/broken/500.json
+        domain = localhost
+        port = ${test.wiremock.mockService.port}
+        allowedRequestHeaders = ["Content-Type"]
+        forceJson = true
+      }
+    }
+    te-pebble {
+      factory = knot
+      config {
+        address = knotx.knot.te.pebble
+        deliveryOptions {
+          sendTimeout = 3000
+        }
+      }
+    }
+  }
+
+  allowInvalidFragmentsHeader = "Allow-Invalid-Fragments"
+  allowInvalidFragmentsParam = "allowInvalidFragments"
```

Comment:
How about reusing `debug` parameter?

Reply:
They exist independently. You can enable debug mode but do not allow invalid fragments to succeed

Reply:
Agree, every consumer can specify when it is enabled. This option is global.